### PR TITLE
New sumo pr

### PR DESCRIPTION
**Pull request by eugenevinitsky**
- upgrade of vehicles and base environments to new sumo
- change bottleneck values for the new capacity diagram
- changed the paths for the sumo setup scripts to point at new binaries
- changed the version number for SUMO

### CODE REVIEW

File: flow/controllers/routing_controllers.py [hunk 1]
```diff
@@ -60,13 +64,15 @@ class GridRouter(BaseRouter):
...
+            return None
+        elif env.vehicles.get_edge(self.veh_id) == \
                 env.vehicles.get_route(self.veh_id)[-1]:
-            new_route = [env.vehicles.get_edge(self.veh_id)]
+            return [env.vehicles.get_edge(self.veh_id)]
         else:
-            new_route = None
-
-        return new_route
+            return None
```
Comment (kanaadp) [inline flow/controllers/routing_controllers.py]: nit, could be combined into a single if statement (first if and else has the same behavior)

File: flow/envs/base_env.py [hunk 2]
```diff
@@ -505,46 +511,18 @@ def step(self, rl_actions):
             self.render()
 
         states = self.get_state()
-        if isinstance(states, dict):
```
Comment (kanaadp) [inline flow/envs/base_env.py]: this is for multiagent right? why was this removed?

Comment (kanaadp) [reply flow/envs/base_env.py]: oh i see, new multiagent base class. These changes should be in a separate PR imo

File: flow/envs/base_env.py [hunk 3]
```diff
@@ -709,24 +697,12 @@ def reset(self):
...
-            # environment class used
-            self.state = np.asarray(states).T
+        # collect information of the state of the network based on the
+        # environment class used
+        self.state = np.asarray(states).T
 
-            # observation associated with the reset (no warm-up steps)
-            observation = np.copy(states)
+        # observation associated with the reset (no warm-up steps)
+        observation = np.copy(states)
```
Comment (kanaadp) [inline flow/envs/base_env.py]: these changes as well should be in a separate PR

**Comment by kanaadp** [comment]
This PR is gonna break travis builds yeah (wrong sumo version)

**Comment by eugenevinitsky** [comment]
Thanks for the review @kanaadp!

**Comment by eugenevinitsky** [comment]
New capacity diagram: 
image
